OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -65,9 +65,43 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
 						var client = orderManager.LobbyInfo.ClientWithIndex(clientId);
 						if (client != null)
 							client.State = Session.ClientState.Disconnected;
+
 						break;
 					}
 
+				case "NewAdmin": /* Find a new admin and hand bots over mid-game. */
+					{
+						var client = orderManager.LobbyInfo.ClientWithIndex(clientId);
+						if (client == null)
+							break;
+
+						if (client.IsAdmin)
```

Comment:
You could move this up into the `null` check above, return early and save one level of indentation:
```
if (client == null || !client.IsAdmin)
	break;
```